zio/core/src/main/scala/tofu/zioInstances/ZioInstances.scala

```diff
@@ -61,7 +61,10 @@ private[zioInstances] class ZioInstances {
   final def zioTofuWithRunInstance[R, E]: ZioTofuWithRunInstance[R, E] =
     zioTofuWithRunInstanceAny.asInstanceOf[ZioTofuWithRunInstance[R, E]]
 
-  final def zioTofuUnliftHasInstance[R <: Has[_], E, C: Tag]: ZioTofuUnliftHasInstance[R, E, C] =
+  final def zioTofuUnliftHasInstance[R <: Has[_], R1 <: Has[_], E, C: Tag](implicit
```

Comment:
That breaks at least my code with all the type arguments filled.
I wonder if I can just remove those, still, maybe it worth add another 5-ary method.

Reply:
Yes, you're right. Fixed